<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(214, 43, 52, 255), stop:1 rgba(251, 175, 178, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>71</y>
      <width>600</width>
      <height>338</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);
border-radius: 20px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>26</y>
       <width>210</width>
       <height>73</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/assets/ISTD Logo.png</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="spnEvent">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>123</y>
       <width>275</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QComboBox" name="spnGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>154</y>
       <width>275</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>185</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 1px solid</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfirm">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>237</y>
       <width>275</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/assets/button-bg.png);
border-radius: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeyboard">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>180</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/keyboard.ico</normaloff>:/images/assets/keyboard.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>27</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22;
background-color: rgb(154, 154, 154);
color: #CCCCCC</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
